aerogear/unifiedpush-operator
# AEROGEAR-9434 use ImageStreams

JIRA: https://issues.jboss.org/browse/AEROGEAR-9434

Use ImageStreams for UPS and its deps (OauthProxy, Postgres).
Also switched from plain Kube Deployments to DeploymentConfigs.

Verification:
* You can deploy the prebuilt image: aliok/ups-opr-jun25-4
* Deploy the operator
* Create unifiedpush CR
* Check the created DeploymentConfigs, Pods, ImageStreams in OpenShift Console
* Delete the CR
* Check if all the resources above are deleted.

Gonna squash my commits after approval.
